Features/Lookuptable/testPages/data/lookup_table_page.py

```diff
@@ -0,0 +1,771 @@
+        self.wait_to_click(self.manage_tables_link)
+        self.wait_to_click(self.edit_state_table)
+        self.wait_for_element(self.select_column)
+        self.click(self.select_column)
+        self.wait_for_element(self.savestate)
+        self.wait_to_click(self.savestate)
+        time.sleep(3)
+        self.wait_to_click(self.applications_menu_id)
+        self.wait_to_click(self.application)
+        self.wait_to_click(self.Ecase_list)
+        self.wait_to_click(self.grid)
+        self.wait_to_click(self.savedbutton)
+        time.sleep(5)
+        assert self.is_present_and_displayed(self.errormessage)
+        print("warning message is displayed")
+
+    def language_selection(self,value):
+        self.wait_to_click(self.settings)
+        self.select_by_text(self.app_language, value)
+        self.wait_to_click(self.done)
+
+    def submit_form_on_registration(self,value,user):
+        self.wait_for_element(self.home)
+        self.wait_to_click(self.home)
+        self.wait_to_click(self.login_user)
+        time.sleep(2)
+        self.wait_to_click((By.XPATH, self.select_user.format(user)))
+        time.sleep(2)
+        self.wait_for_element(self.login)
+        self.wait_to_click(self.login )
+        time.sleep(2)
+        self.language_selection(value)
+        self.wait_for_element(self.sync)
+        self.wait_to_click(self.sync)
+        time.sleep(20)
+        self.wait_for_element(self.start)
+        self.wait_to_click(self.start)
+        self.wait_for_element(self.inapp_caselist)
+        self.wait_to_click(self.inapp_caselist)
+        self.wait_for_element(self.inapp_registrationform)
+        self.wait_to_click(self.inapp_registrationform)
+        time.sleep(3)
+        if user != 'appiumtest':
+            self.wait_to_click(self.inapp_selectoption)
+            time.sleep(3)
+            value = self.get_text(self.inapp_selectoption)
+            self.wait_to_click(self.inapp_next)
+            time.sleep(3)
+            self.wait_to_click(self.inapp_continue)
+            #self.send_keys(self.question_display_text, self.question_display_text_name)
+            time.sleep(5)
+            assert self.is_present_and_displayed(self.successmsg)
+            print("form submitted succesfully:", value)
+            en = "Uttar Pradesh"
+            hin = "उत्तर प्रदेश"
+            if value == hin:
+                print("Form submitted in HINDI")
+            elif value == en:
+                print("Form submitted in ENGLISH")
+        else:
+            print("Make sure the 'Inapp' lookup table is available, and that its contents are accessible to the current user.")
+
+
+
+    def language_check(self):
+        self.clear(self.value_field)
+        self.wait_to_click(self.value_field)
+        self.send_keys(self.value_field, "id")
+        self.clear(self.display_field)
+        self.wait_to_click(self.display_field)
+        self.send_keys(self.display_field, "name[@lang = jr:itext('lang-code-label')]" )
```

Comment:
assign this value to some variable within the constructor or inside the UserData class and then call that variable as a parameter